lib/python/qmk/cli/generate/config_h.py

```diff
@@ -13,6 +13,14 @@
 from qmk.constants import GPL2_HEADER_C_LIKE, GENERATED_HEADER_C_LIKE
 
 
+def generate_define(define, value=None):
+    value = f' {value}' if value else ''
```

Comment:
```suggestion
    value = f' {value}' if value is not None else ''
```
Just guards against the python value resolving to a False condition.